plugins/modules/ipaautomember.py

```diff
@@ -153,7 +153,7 @@ def gen_condition_args(grouping,
     _args = {}
     if grouping is not None:
         _args['type'] = grouping
-    if key is not None:
+    if key is not None:2.12
```

Comment:
This needs to be fixed.

Reply:
Apologies, looks like that was accidentally added.